Engines/Wine/Verbs/dotnet20sp2/script.js

```diff
@@ -55,6 +55,9 @@ Wine.prototype.dotnet20sp2 = function () {
     }
 
     this.windowsVersion(osVersion);
+    this.overrideDLL()
+        .set("", dlls)
```

Comment:
This is dangerous: you could remove other overrides !

Reply:
Unlikey, the `dlls` variable have only these overrides in it.

Reply:
And what if other people have override dlls before installing the verb ?

Reply:
Ok sorry, I have seen now what you have done. Clever way ^^ :+1: